### ISSUE TITLE
Wrong gender information

### ISSUE THREAD

**Issue by frankenjoe** [audeering/emodb#2]
With `1.1.0` the gender information for some speakers changed:

* Version `1.0.1`:

```
      3: {gender: male, age: 31, language: German}
      8: {gender: female, age: 34, language: German}
      9: {gender: male, age: 21, language: German} # Wrong
      10: {gender: female, age: 32, language: German} # Wrong
      11: {gender: male, age: 26, language: German}
      12: {gender: male, age: 30, language: German}
      13: {gender: female, age: 32, language: German}
      14: {gender: female, age: 35, language: German}
      15: {gender: male, age: 25, language: German}
      16: {gender: female, age: 31, language: German}
```

* Version `1.1.0`:

```
      3: {gender: male, age: 31, language: deu}
      8: {gender: female, age: 34, language: deu}
      9: {gender: male, age: 21, language: deu} # Wrong
      10: {gender: female, age: 32, language: deu} # Wrong
      11: {gender: male, age: 26, language: deu}
      12: {gender: female, age: 30, language: deu} # Wrong
      13: {gender: male, age: 32, language: deu} # Wrong
      14: {gender: female, age: 35, language: deu}
      15: {gender: male, age: 25, language: deu}
      16: {gender: female, age: 31, language: deu}
```

Interestingly, both versions contain wrong labels :)

**Comment by frankenjoe** [comment]
Here's the correct mapping from the original site: http://emodb.bilderbar.info/index-1024.html

![Uploading image.png…]()

**Comment by frankenjoe** [comment]
Fixed in #3 

I updated the benchmark to the new version and now we a 100% recall for the best model.
